lib/Raven/Configuration.php

```diff
@@ -740,10 +706,11 @@ private function configureOptions(OptionsResolver $resolver)
         $resolver->setAllowedTypes('trust_x_forwarded_proto', 'bool');
         $resolver->setAllowedTypes('prefixes', 'array');
         $resolver->setAllowedTypes('serialize_all_object', 'bool');
+        $resolver->setAllowedTypes('serializer', SerializerInterface::class);
+        $resolver->setAllowedTypes('reprSerializer', SerializerInterface::class);
```

Comment:
This option name should follow the lowercase underscore naming convention used by all other options